src/Logger.ts

```diff
@@ -22,7 +22,7 @@ export class Logger {
         logLevel?: LogLevel,
         public prefix?: string
     ) {
-        this.logLevel = logLevel;
+        this.logLevel = logLevel ?? LogLevel.log;
```

Comment:
This is a little subtle. If `undefined` is passed for `logLevel`, then the setter will be called with `undefined`. The setter defines the type as `LogLevel` and I don't want to change that signature. Internally the setter checks whether its argument is a string; if it's not it falls back to `LogLevel.log`. So this duplicates that "default log level" and sets it in the constructor.

I didn't remove the "is this a string" check from the setter because I don't know whether other code in the system might set the log level to something other than a string.